images/macos/software-report/SoftwareReport.Browsers.psm1

```diff
@@ -49,4 +51,27 @@ function Get-FirefoxVersion {
 
 function Get-GeckodriverVersion {
     return Run-Command "geckodriver --version" | Select-Object -First 1
-}
\ No newline at end of file
+}
+
+function Build-BrowserWebdriversEnvironmentTable {
+    Write-Host "Build-BrowserWebdriversEnvironmentTable"
+    return @(
+        @{
+            "Name" = "CHROMEWEBDRIVER"
+            "Value" = $env:CHROMEWEBDRIVER
+        },
+        @{
+            "Name" = "EDGEWEBDRIVER"
+            "Value" = $env:EDGEWEBDRIVER
+        },
+        @{
+            "Name" = "GECKOWEBDRIVER"
+            "Value" = $env:GECKOWEBDRIVER
+        }
+    ) | ForEach-Object {
+        [PSCustomObject] @{
+            "Name" = $_.Name
+            "Value" = Get-PathWithLink($_.Value)
```

Comment:
These variables are not symlinks so we don't need to use `Get-PathWithLink` here